engine/security/rootless.md

```diff
@@ -50,72 +50,71 @@ testuser:231072:65536
 
 > Note: We recommend that you use the Ubuntu kernel.
 
-#### Ubuntu
-
+<ul class="nav nav-tabs">
+  <li class="active"><a data-toggle="tab" data-target="#hint-ubuntu">Ubuntu</a></li>
+  <li><a data-toggle="tab" data-target="#hint-debian">Debian GNU/Linux</a></li>
+  <li><a data-toggle="tab" data-target="#hint-arch">Arch Linux</a></li>
+  <li><a data-toggle="tab" data-target="#hint-opensuse">openSUSE</a></li>
+  <li><a data-toggle="tab" data-target="#hint-centos8-and-fedora">CentOS 8 and Fedora</a></li>
+  <li><a data-toggle="tab" data-target="#hint-centos7">CentOS 7</a></li>
```

Comment:
Given that we don't ship packages for Arch Linux or openSUSE, perhaps we should consider removing those instructions. At least, it may be good to move them as last tabs (and perhaps order the tabs base on "most common")

We can do that in a follow-up though; I see some other possible improvements, so may reserve some time to do a follow up (I'll ask you to review those 👍)